application/auth/helpers.py

```diff
@@ -28,31 +28,29 @@
 SERVER_ERROR_MSG = 'Authorization failed. Please contact support.'
 
 
-def save_user():
-    user = User.query.filter_by(email=session.get("email")).first()
+def save_user(payload):
+    user = User.query.filter_by(email=payload.get("email")).first()
     if not user:
```

Comment:
What response do you give when the user is not there?

Reply:
What do you mean? At this stage, I save the user to the db if they don't already exist in the database